### DISCUSSION TITLE
171

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#810] [Comments]
http://jojoldu.tistory.com/171

**Comment by jojoldu** [comment]
작성자: 우끼리

application.yml에
"
profiles:
  inclulde:google
"
하는게 더 간결하지 않을까요?

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

음.. 말씀하신게 spring.profiles.include 이 맞으신가요?
해당 옵션은 포함시킬 Profile (ex: dev, real 등) 을 추가하는 옵션이라 google.yml을 호출하기 위해선 추가작업이 필요할것 같습니다
(예를들어 google profile을 만들고 google profile일 경우에는  google.yml을 호출해야하는등? )
말씀하신대로 옵션 추가해서 실행하면 동일하게 google.yml을 못불러오는 에러가 발생합니다.
혹시나 제가 테스트 한 방법이 잘못되었다면 답변부탁드리겠습니다^^;

**Comment by jeongmincha** [comment]
너무 옛날 글이라 답변은 어려우시겠지만;;
둘 중에 어떤 해결책을 선택하셨었나요?
두 해결책에 대한 트레이드오프가 어떻게 될지, 그 중에서 어떤 장/단점이 크게 다가왔는지 등의 의견을 듣고 싶습니다.
